include/highfive/bits/H5Dataspace_misc.hpp

```diff
@@ -125,100 +125,16 @@ inline std::vector<size_t> DataSpace::getMaxDimensions() const {
     return details::to_vector_size_t(maxdims);
 }
 
-template <typename ScalarValue>
-inline DataSpace DataSpace::From(const ScalarValue& scalar) {
-    (void)scalar;
-    static_assert(
```

Comment:
As much as I like simplifying, the assert message could be helpful, better than a whole lot of template errors.

Reply:
What others think of it?

Reply:
Having a `static_assert` is nice, but following my earlier comment for maintainability when new types are added it would be nice if it follows from the `inspector` (not sure have that could be done though?).

Reply:
I disagree with static_assert here ^^